annyang.js

```diff
@@ -113,9 +113,9 @@
         if (result) {
           var parameters = result.slice(1);
           if (debugState) {
-            console.log('command matched: %c'+currentCommand.originalPhrase, debugStyle);
+            logMessage('command matched: %c'+currentCommand.originalPhrase);
             if (parameters.length) {
-              console.log('with parameters', parameters);
+              logMessage('with parameters "'+parameters.join('", "')+'"');
```

Comment:
This changes the API. What was previously logged a certain way, is now logged differently.

I would personally think many times before doing a breaking change like that. It may be an edge case, but to some users, it may break their code (or at the very least, their tests).